### PR TITLE
1) Laura CodeBat2 done. 2) Also added Nitrous commit. 3) Made my Simple post.

### PR DESCRIPTION
**Pull request by Lasher09**
Here it is:
1) http://lasher09.github.io/spring2014/2014/02/16/Laura-codBat2.html

2) Also added my Nitrous push post.

3) Made my Simple post. Link: http://silshack-githublasher09.herokuapp.com/

### CODE REVIEW

File: _posts/2014-02-16-Laura-codBat2.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Here is the picture
+![CodingBat2 Screenshot](http://puu.sh/6Z2b3.png "CodingBat2")
+
+##Interesting 1
+###Python > String-2 > end_other 
+Question: Given two strings, return True if either of the strings appears at the very end of the other string, ignoring upper/lower case differences (in other words, the computation should not be "case sensitive"). Note: s.lower() returns the lowercase version of a string. 
+
+With all the adding and slicing going on in this series, it was nice to use some simple code.
+```
```
Comment (eah13) [inline _posts/2014-02-16-Laura-codBat2.md:17]: You need an empty line above each code block to set them off.  Check your preview site a second time.

**Comment by eah13** [comment]
@Lasher09 take a look at these code blocks; need to fix em before I can merge.

File: _posts/2014-01-16-turtledrawing.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 author: laura
-title: Laura’s Turtle Exercise: a silly drawing
+title: Laura’s Turtle Exercise a silly drawing
```
Comment (eah13) [inline _posts/2014-01-16-turtledrawing.md:4]: Awesome- thanks for fixing this.

File: _posts/2014-02-05-Laura-testint-nitrous.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+author: laura
+title: Testing Nitrous
+date: 2014-02-19
+---
+
+## Hi!
+
+{% include python %}
+
+print ("Hi I am python code.")
```
Comment (eah13) [inline _posts/2014-02-05-Laura-testint-nitrous.md:12]: Why, hello there!

File: _posts/2014-02-16-Laura-codBat2.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  a = a.lower()
+  b = b.lower()
+  return (b.endswith(a) or a.endswith(b))
+```
+
+##Interesting 2
+###Python > List-2 > big_diff 
+Question: Given an array length 1 or more of ints, return the difference between the largest and smallest values in the array. Note: the built-in min(v1, v2) and max(v1, v2) functions return the smaller or larger of two values. 
+
+Again, simplicity rules.
```
Comment (eah13) [inline _posts/2014-02-16-Laura-codBat2.md:28]: I dig it.

File: _posts/2014-02-16-Laura-codBat2.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+def big_diff(nums):
+  return max(nums) - min(nums)
+```
+
+##Interesting 3
+###Python > List-2 > count_evens 
+Question: Return the number of even ints in the given array. Note: the % "mod" operator computes the remainder, e.g. 5 % 2 is 1. 
+
+This helped me understand getting to even and odd elements in a string
```
Comment (eah13) [inline _posts/2014-02-16-Laura-codBat2.md:39]: Awesome!  Nicely done.

**Comment by Lasher09** [comment]
What need fixed?
[image: Inline image 1]

Does it need the date?

Thanks.

**Comment by eah13** [comment]
@Lasher09 I think this is my fault for not merging when you were ready to.  Open an issue that we can tag to the Simplehack exercise if you don't mind.

Nice work!

**Pull request merged**